hack/test/capi.sh

```diff
@@ -24,10 +24,6 @@ cat ${PWD}/hack/test/manifests/capg-components.yaml| envsubst > ${TMP}/capg-comp
 ##They've got an incorrect image pull policy.
 ##curl -L ${CAPG_COMPONENTS} | envsubst > ${TMP}/capg-components.yaml
 
-## Clean up osctl endpoint, fetch kubeconfig
-e2e_run "osctl config endpoint 10.5.0.2"
```

Comment:
this line is required I believe, as it switches endpoint from 127.0.0.1 (when running outside of docker network) to 10.5.0.2 (when running inside docker network)

Reply:
This seemed to work without this.

Reply:
Do you know when we needed this? For CI? Or local?

Reply:
For CI. `basic-integration` should leave the address in config in a way it works for pod-to-pod communication (from build step pod to docker service pod), while e2e uses`docker run`, so it runs inside a docker service pod and inside a docker network, and pod IP won't be accessible there iirc.

Reply:
Ok. Doesn't hurt to put it back. I will do that 👍.